digdag-server/src/main/java/io/digdag/server/Authenticator.java

```diff
@@ -57,6 +59,19 @@ default boolean isAccepted()
         String getErrorMessage();
 
         Optional<Config> getUserInfo();
+
+        Optional<Map<String, String>> getSecrets();
```

Comment:
I think it's better to make this `Optional<Supplier<Map<String, String>>>`. With `Optional<Map>`, Authenticator always needs to fetch security information from somewhere even when controller doesn't need them.